### PR TITLE
MIME registration +typo fix - (closes #68)

### LINKED ISSUE #68: Serving manifest should be defined as a MIME registration
The section [Serving Manifests](http://runtime.sysapps.org/#serving-manifests) should really be defined in terms of a MIME type registration. This type will need to be registered with IANA.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Addresses https://github.com/sysapps/runtime/issues/68

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1106,6 +1106,189 @@
...
+              described in [[!JSON]] and [[!UTR36]] apply. In addition,
+              implementers need to impose their own implementation-specific
+              limits on the values of otherwise unconstrained member types,
+              e.g. to prevent denial of service attacks, to guard against
+              running out of memory, or to work around platform-specific
+              limitations.
+            </p>
+            <p>
+              The manifest document allows authors, through the permissions and
+              required_features, to request permission to enable security
```
Comment (mounirlamouri) [inline index.html:1174]: required_features is not really to enable features, just to filter out some applications from the stores

File: index.html [hunk 2]
```diff
@@ -1106,6 +1106,189 @@
...
+              unpacking of zip packages into the file system might lead to
+              undesirable and security relevant effects, such as overwriting of
+              system files.
+            </p>
+          </dd>
+          <dt>
+            Applications that use this media type:
+          </dt>
+          <dd>
+            Web browsers
```
Comment (mounirlamouri) [inline index.html:1230]: Should we add Web Runtimes?

**Comment by mounirlamouri** [comment]
I'm merging this but please consider my comments as follow-ups.

File: index.html (hunk 2 above)
Comment (marcoscaceres) [inline index.html:1230]: I don't think we should add Web Runtimes. I think that will just confuse people. A web runtime is a kinda web browser ... or uses a lot of web browser parts, so I think one can argue that they are kinda the same.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html:1174]: Filed bug: https://github.com/sysapps/runtime/issues/88

**Pull request merged**